specifyweb/frontend/js_src/lib/components/QueryBuilder/FieldFilter.tsx

```diff
@@ -199,7 +202,7 @@ export function QueryInputField({
       className={`
         relative min-w-[theme(spacing.40)] after:invisible
         after:block after:px-2 after:leading-[0px] after:content-[attr(data-value)]
-      `}
+      ${isBasic ? 'flex-1' : ''}`}
```

Comment:
can you indent this correctly?
(indent by 1 to match the lines before it, and also put a new line:
```ts
    ${isBasic ? 'flex-1' : ''}
  `}
```